components/mpas-ocean/driver/ocn_comp_mct.F

```diff
@@ -790,7 +790,10 @@ end subroutine xml_stream_get_attributes
     call t_stopf ('mpaso_mct_init')
 
     call mpas_pool_get_config(domain % configs, 'config_land_ice_flux_mode', config_land_ice_flux_mode)
-    if ( trim(config_land_ice_flux_mode) .eq. 'pressure_only' ) then
+    if ( trim(config_land_ice_flux_mode) .eq. 'off' ) then
+       call seq_infodata_PutData( infodata, ocn_prognostic=.true., ocnrof_prognostic=.true., &
+                                            ocn_c2_glcshelf=.false.)
+    elseif ( trim(config_land_ice_flux_mode) .eq. 'pressure_only' ) then
        call seq_infodata_PutData( infodata, ocn_prognostic=.true., ocnrof_prognostic=.true., &
                                             ocn_c2_glcshelf=.false.)
     else if ( trim(config_land_ice_flux_mode) .eq. 'standalone' ) then
```

Comment:
Unless I'm missing something, these 3 entries re identical.

```suggestion
    if ( trim(config_land_ice_flux_mode) .eq. 'off' .or. &
            trim(config_land_ice_flux_mode) .eq. 'pressure_only'.or. &
            trim(config_land_ice_flux_mode) .eq. 'standalone') then
```

Reply:
Alternatively, I can make this change in https://github.com/E3SM-Ocean-Discussion/E3SM/pull/45.

Reply:
Fixed in https://github.com/E3SM-Project/E3SM/pull/5464/commits/aa144a63b3e2d2efbaba6f298ef5696df51397f4